### ISSUE TITLE
Map rotation when routing and not moving

### ISSUE THREAD

**Issue by TuomLarsen** [organicmaps/organicmaps#6094]
**Is your feature request related to a problem? Please describe.**
When I'm on the bike and using routing (i.e. navigation), "up" can be either the direction of travel or the north of the map. The problem is, when I stop, for instance to check the route on the phone hidden in a pocket, it is sometimes confusing to know which direction to go as now, when I rotate the phone in my hand the map stays static (as opposed when "not routing").

**Describe the solution you would like**
A setting option to set what kind of "up" one wants during routing: "north" (does not rotate map), direction of travel (rotates the map), phone's compass (always rotates map)

**Describe alternatives you have considered**
Stop routing, rotate the map to get a sense of surrounding, turn on routing back again.

**Additional context**
To put it differently, what I'm after is to have the same behaviour of the map rotation while routing as the one when it is not routing.

**Comment by biodranik** [comment]
To clarify: you're talking only about an _active_ navigation mode, right?

**Comment by TuomLarsen** [comment]
@biodranik Yes.

**Comment by crgmrgn** [comment]
Related, this is an obviously missing feature when using navigation under Carplay, quite off-putting if you are conditioned to "direction of travel" type display.

**Comment by richlv** [comment]
This might be impossible with solving #5898 first.
